src/gei/GithubSecretScanningAlertServiceFactory.cs

```diff
@@ -0,0 +1,15 @@
+using Octoshift;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter;
+
+public sealed class GitHubSecretScanningAlertServiceFactory: ISecretScanningAlertServiceFactory
+{
+    private readonly OctoLogger _octoLogger;
+
+    public GitHubSecretScanningAlertServiceFactory(OctoLogger octoLogger)
+    {
+        _octoLogger = octoLogger;
+    }
```

Comment:
you can inject the Github API factories and then use them in the `Create` method:

```c#
public GitHubSecretScanningAlertServiceFactory(OctoLogger octoLogger, ISourceGithubApiFactory sourceGithubApiFactory, ITargetGithubApiFactory targetGithubApiFactory)
{
    _octoLogger = octoLogger;
    _sourceGithubApiFactory = sourceGithubApiFactory;
    _targetGithubApiFactory = targetGithubApiFactory;
}
```